analyzer/src/test/kotlin/managers/utils/NpmDependencyHandlerTest.kt

```diff
@@ -94,9 +94,11 @@ private fun createIdentifier(name: String): Identifier =
  */
 private fun createModuleInfo(
     id: Identifier,
-    packageFile: File = File("package.json"),
+    packageFile: File = File("project/package.json"),
     dependencies: Set<NpmModuleInfo> = emptySet()
-): NpmModuleInfo = NpmModuleInfo(id, packageFile, dependencies)
+): NpmModuleInfo {
+    return NpmModuleInfo(id, packageFile.parentFile, packageFile, dependencies)
+}
```

Comment:
Why also this change?

Reply:
I.e. why not keeping it a function expression?

Reply:
Debug reasons :)